Sendgrid Cloud Extract documentation (#524)
rudderlabs/rudderstack-docs: cloud-extract-sources/destination-settings.mdx

---
title: "Destination Settings"
description: >-
 Detailed technical description of the different destinations settings for configuring the Cloud Extract sources.
---

# Destination Settings

This guide details the common destination settings to be configured while setting up your Cloud Extract sources.

These settings are:

- Table prefix

> - Sync schedule settings

Reviewer: ```suggestion
- Schedule settings
```
As displayed in the screenshot

<img src="../assets/cloud-extract-sources/sendgrid-connection-settings-2.png" alt="Sendgrid destination settings" />

## Table prefix

> RudderStack uses your specified table prefix to create a table in your data warehouse and load all your Sendgrid data into it.

Reviewer: ```suggestion
RudderStack uses your specified table prefix to create a table in your data warehouse and load all the destination data into it.
```

Author: actually it should be the source data, i.e. data from the cloud extract source

Author: tweaked your suggestion accordingly.

<div class="infoBlock">

If you do not specify this field, RudderStack sets the source name as the table name, by default.
</div>

> ## Sync schedule

Reviewer: ```suggestion
## Schedule settings
```
